lib/Test2/Compare/Number.pm

```diff
@@ -33,6 +35,14 @@ sub name {
     return $in;
 }
 
+my %NEGATED = (
+    '==' => '!=',
```

Comment:
Does this need a line
```perl
    '!=' => '==',
```
?

Reply:
Hrmmmm... possibly... I didn't implement having the mode be `!=` but might as well for consistency.

Reply:
456dc284